packages/autorest.python/autorest/codegen/serializers/builder_serializer.py

```diff
@@ -1440,6 +1440,7 @@ def get_long_running_output(self, builder: LROOperationType) -> List[str]:
                 retval.append("    response_headers = {}")
             if (
                 not self.code_model.options["models_mode"]
+                or self.code_model.options["models_mode"] == "dpg"
```

Comment:
we need the logic otherwise the generated code will be like:
![image](https://user-images.githubusercontent.com/70930885/192700640-46eaa54c-0173-4bf2-9fea-aec260ea11a2.png)